src/components/courselayout.tsx

```diff
@@ -0,0 +1,20 @@
+interface CourseLayoutProps {
+  title: string;
+  description: string;
+}
+
```

Comment:
props do not need to be passed here, sorry for having unclear issues but we'll replace the lorem ipsum/description with actual text soon but you can just put lorem ipsum text for now